cypress/support/commands.js

```diff
@@ -30,33 +30,40 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy^="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
-  });
-});
+Cypress.Commands.add(
+  'register',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/api/users', {
+      user: {
+        email,
+        username,
+        password,
+      },
+    });
+  }
+);
```

Comment:
The 'register' command does not include any assertions to verify the success of the registration request. Consider adding assertions to ensure that the registration was successful.